### DISCUSSION TITLE
Subsequent deploys not building packages

### DISCUSSION THREAD

**Discussion by mmJordanO** [vercel/turborepo#427] [Help]
I refactored my root level package.json to utilize the pipeline.  Before I was doing some manual yarn builds in the scripts.  This new way has it build properly on my local machine and, if I run a dev server, also runs.  Effectively it needs to build some packages before trying to run the apps.

The first time I deployed this test to Vercel it worked fine, however after updating the repo it's no longer working.  It seems like it's getting cache hits where it shouldn't (i.e. the package hasn't been built) and thus the necessary packages don't get built and thus not found.

Relevant scripts in root's `package.json` :

```
"scripts": {
     "deploy": "turbo run build --scope=landing-pages --includeDependencies --no-deps",
}

"turbo": {
    "pipeline": {
      "build": {
        "outputs": [
          "dist/**",
          ".next/**"
        ],
        "dependsOn": [
          "^build"
        ]
      },
      "lint": {
        "outputs": []
      },
      "deploy": {
        "dependsOn": [
          "^build"
        ]
      },
      "dev": {
        "cache": false,
        "dependsOn": [
          "^build"
        ]
      }
    }
  },

```

Is there a way for a package to detect if has indeed been built?

**Comment by anthonyshew** [comment]
Seeing the use of several deprecated flags on this one so going to mark as outdated!
